modules/installation/pages/hw-and-sw-requirements.adoc

```diff
@@ -249,3 +126,137 @@ In an on-premises installation, the system is fully functional without a web bro
+
+The sizing recommendations below apply to each server node. If you have more than a several hundred GBs of data, you should consider deploying a cluster of multiple nodes, to distribute your data.
+
+NOTE: Consult a TigerGraph solution architects for an estimate of memory and storage needs.
+
+=== Personal Use
+Example profile: A single user on a personal computer (desktop or laptop), who is learning to use the software and to write applications.
+|===
+| Component | Specification 
+
+| CPU
+| 4 cores (≥ 2.10GHz)
+
+| RAM
+| 8 GB
+
+| Storage
+| ≥ 50 GB (SSD)
+
+| Network
+| 1 Gigabit Ethernet adapter
+|===
+
+=== Development, UAT, or SIT System
+|===
+| Component | Specification 
+
+| CPU
+| ≥ 16 cores (≥ 3.00GHz)
+
+| RAM
+| ≥ 32 GB
+
+| Storage
+| ≥ 300 GB (SSD)
+
+| Network
+| 1 Gigabit Ethernet adapter
+|===
+
+=== Production System
+NOTE: Production systems should consider having replica clusters for redundancy and availability.
+|===
+| Component | Specification 
+
+| CPU
+| ≥ 32 cores (≥ 3.40GHz)
+
+| RAM
+| ≥ 64 GB
+
+| Storage
+| ≥ 500 GB (SSD)
+
+| Network
+| 10 Gigabit Ethernet adapter
+|===
+
+
+=== Additional Considerations
+This section descriptions considerations beyond sizing for the CPU, memory, and storage.
+
+==== CPU
+
+Choosing the right CPU model is important.
+For example, *Intel Core i5* may be good enough for development or certain test systems where latency and throughput are not critical. For production systems, we suggest higher grade CPUs such as *Intel Core i7*, *Intel Xeon Gold*, and *AMD Epyc*.
+
+CPU clock speeds are not directly comparable across different processor designs, but within the same processor family, faster is better. We highly recommend using a *speed of 3.00GHz or higher*.
+
+Below is an example of a production hardware configuation.
+```
+Architecture:        x86_64
+CPU op-mode(s):      32-bit, 64-bit
+Byte Order:          Little Endian
+CPU(s):              24
+On-line CPU(s) list: 0-23
+Thread(s) per core:  1
+Core(s) per socket:  1
+Socket(s):           24
+NUMA node(s):        1
+Vendor ID:           GenuineIntel
+CPU family:          6
+Model:               85
+Model name:          Intel(R) Xeon(R) Gold 6248R CPU @ 3.00GHz
+Stepping:            7
+CPU MHz:             2992.969
+```
+
+==== RAM (Memory)
+
+RAM is needed both to _store_ a working copy of our data and to do _computational_ work. TigerGraph is a hybrid in-memory database, meaning that in order to achieve maximum performance, it will attempt to load a copy of your entire working graph into RAM.
+The RAM size is based on multiple factors:
+
+* Type of workload (e.g., OLAP/OLTP queries, intensive CRUD operation....)
+* Query SLAs
+* Dataset partitioning (number of partitions in your TigerGraph cluster)
+
+RAM is not only size but also type. DDR4 is recommended because it gives you significantly faster data transfer rates, better power efficiency, and better data granularity than DDR3.
```

Comment:
This page will slowly become outdated - DDR5 has been out now for more than a year. Suggest simply saying something along the lines of "the faster the data transfer speed of the RAM, the better." This may not even be necessary as the user probably knows that speed is good.

Reply:
I thought about that. It will probably be a few years before DDR5 is considered mainstream instead of premium. I might prefer to say "DDR4 or faster".